Practice1/src/css/footer.css

```diff
@@ -0,0 +1,27 @@
+.page-footer {
+    position: relative;
+    background-color: #003366;
```

Comment:
Should define a color variable and use it here.

Reply:
This section has been closed